### PR TITLE
DOC fix truncated documentation for Series.unstack(), Series. 

### LINKED ISSUE #31235: DOC BUG API Reference documentation truncated for Series.unstack(), Series.
The [API Reference documentation for `Series`](https://pandas.pydata.org/pandas-docs/stable/reference/series.html) and [`DataFrame`](https://pandas.pydata.org/pandas-docs/stable/reference/frame.html) have truncated documentation strings for `Series.at_time()`, `Series.unstack()`, and `DataFrame.at_time()`. The bug appears to occur where the docstring contains ". " (that is, a period followed by a space). Presumably, this pattern is being used to find the end of the one-line function description.

### PR DESCRIPTION
**Pull request by anjalis2112**
fixes https://github.com/pandas-dev/pandas/issues/31235

Changed "e.g. " to "e.g., "
Changed "a.k.a. " to "also known as"
This helps avoid the truncation caused in the docstring

### CODE REVIEW

**Comment by rohitsanj** [comment]
@anjalis2112 congrats on raising your first PR to Pandas! 👍

**Comment by WillAyd** [comment]
No strong objection but I do think this is a little hacky. Do you know where this limitation is being imposed?

**Comment by anjalis2112** [comment]
The issue here is that, every time a ". " (full stop + space) is encountered, it's presumed to be the end of a one line docstring.

**Comment by anjalis2112** [comment]
The issue here is that, every time a ". " (full stop + space) is encountered, it's presumed to be the end of a one line docstring function description.

File: doc/source/user_guide/indexing.rst [hunk 1]
```diff
@@ -105,7 +105,7 @@ Basics
 ------
 
 As mentioned when introducing the data structures in the :ref:`last section
-<basics>`, the primary function of indexing with ``[]`` (a.k.a. ``__getitem__``
+<basics>`, the primary function of indexing with ``[]`` (a.k.a ``__getitem__``
```
Comment (jorisvandenbossche) [inline doc/source/user_guide/indexing.rst]: You don't need to change this in the user guide. The issue only comes up in the first line of docstrings

File: pandas/core/series.py [hunk 2]
```diff
@@ -3389,7 +3389,7 @@ def explode(self) -> "Series":
         See Also
         --------
         Series.str.split : Split string values on specified separator.
-        Series.unstack : Unstack, a.k.a. pivot, Series with MultiIndex
+        Series.unstack : Unstack, a.k.a pivot, Series with MultiIndex
```
Comment (jorisvandenbossche) [inline pandas/core/series.py]: also not needed here

File: pandas/core/series.py [hunk 3]
```diff
@@ -3432,7 +3432,7 @@ def explode(self) -> "Series":
 
     def unstack(self, level=-1, fill_value=None):
         """
-        Unstack, a.k.a. pivot, Series with MultiIndex to produce DataFrame.
+        Unstack, a.k.a pivot, Series with MultiIndex to produce DataFrame.
```
Comment (jorisvandenbossche) [inline pandas/core/series.py]: Maybe just reword this to use "or" instead of "a.k.a.", or wite it in full? "also known as"

Comment (anjalis2112) [reply pandas/core/series.py]: Alright! Will do that

**Review by WillAyd — APPROVED** [review]

**Comment by WillAyd** [comment]
Thanks @anjalis2112
